src/core_plugins/kibana/public/management/sections/objects/_view.html

```diff
@@ -1,36 +1,127 @@
-<kbn-management-app section="kibana">
-  <kbn-management-objects-view class="container">
-    <div class="pull-right" style="margin-top: 20px;">
-      <a href="{{ link }}" class="btn btn-primary">View {{ title }}</a>
-      <a confirm-click="delete()" class="btn btn-danger"><i class="fa fa-trash-o"></i> Delete {{ title }} Object</a>
+<kbn-management-app section="kibana" class="kuiView">
+  <kbn-management-objects-view class="kuiViewContent kuiViewContent--constrainedWidth">
+    <!-- Header -->
+    <div class="kuiViewContentItem kuiSubHeader">
+      <h1 class="kuiTitle">
+        Edit {{ title }}
+      </h1>
+
+      <div class="kuiButtonGroup">
+        <a
+          class="kuiButton kuiButton--basic kuiButton--iconText"
+          href="{{ link }}"
+        >
```

Comment:
A better example just occurred to me. I used to write a lot of Jade and that must be where I picked up this pattern. Here's how [Jade (now known as Pug)](https://pugjs.org/language/attributes.html) structures its tags:

```jade
input(
  type='checkbox'
  name='agreement'
  checked
)
```

Can you see how this structure is much more analogous to the way I'm proposing? The tag is almost like a function, and the attributes are what gets passed into it. The brackets act to encapsulate the attributes... due to the way HTML works though, your opening bracket comes before the "function name" instead of after.